README.md

```diff
@@ -95,15 +91,14 @@ The following command will scan the directory `$TXT_DIR` looking for .txt files
 mvn exec:java -Dexec.mainClass=phrasecount.cmd.Load -Dexec.args="fluo.properties $TXT_DIR" -Dexec.classpathScope=test
```

Comment:
I made bash scripts in c70a12d and updated the README.  I did not make the bash scripts check usage, as the java program does this.